frontend/src/components/features/conversation-panel/conversation-card.tsx

```diff
@@ -240,22 +240,65 @@ export function ConversationCard({
         title="Metrics Information"
         testID="metrics-modal"
       >
-        <div className="space-y-2">
-          {metrics?.cost !== null && (
-            <p>Total Cost: ${metrics.cost.toFixed(4)}</p>
-          )}
-          {metrics?.usage !== null && (
-            <>
-              <p>Tokens Used:</p>
-              <ul className="list-inside space-y-1 ml-2">
-                <li>- Input: {metrics.usage.prompt_tokens}</li>
-                <li>- Output: {metrics.usage.completion_tokens}</li>
-                <li>- Total: {metrics.usage.total_tokens}</li>
-              </ul>
-            </>
+        <div className="space-y-4">
+          {(metrics?.cost !== null || metrics?.usage !== null) && (
+            <div className="rounded-md p-3">
+              <div className="grid gap-3">
+                {metrics?.cost !== null && (
+                  <div className="flex justify-between items-center border-b border-neutral-700 pb-2">
+                    <span className="text-lg font-semibold">Total Cost:</span>
```

Comment:
This is very tiny, but just so you know, some other areas in the world also use "$" sign (e.g. Hong Kong Dollars) so this could be confusing :)

```suggestion
                    <span className="text-lg font-semibold">Total Cost (USD):</span>
```